.github/json_matrices/engine-matrix.json

```diff
@@ -1,4 +1,9 @@
 [
+    {
+        "type": "valkey",
+        "version": "9.0.0-rc1",
+        "run": "always"
```

Comment:
We should probably have only 2 with "always", but I'm ok with this change if it is temporary